tensorflow/compiler/tf2tensorrt/convert/trt_optimization_pass.cc

```diff
@@ -232,8 +244,15 @@ Status TRTOptimizationPass::Optimize(grappler::Cluster* cluster,
         UpdateFunctionSpecificConversionParams(params_, func_item.func_attr()));
   }
 
-  return ConvertGraph(params_, optimized_item, nodes_to_preserve, cluster,
+  auto status = ConvertGraph(params_, optimized_item, nodes_to_preserve, cluster,
```

Comment:
We don't use "auto" in this case. see [C++ style guide](https://google.github.io/styleguide/cppguide.html#Type_deduction)